### ISSUE TITLE
Visual Studio shows bright line below tabs

### ISSUE THREAD

**Issue by Binaryify** [microsoft/theme-converter-for-vs#145]
**Describe the bug**
after covert the theme, Visual Studio shows bright line below tabs and I also see that in official themes

**Screenshots**
![image](https://user-images.githubusercontent.com/12221718/172044670-89635b31-008f-46e0-9144-1aa56cea6c1e.png)

![image](https://user-images.githubusercontent.com/12221718/172044698-5b470ea7-79d4-4cfb-ab9e-8770f1e0281f.png)

How to avoid that

**Comment by fgimian** [comment]
Just referencing the original issue where I reported this too https://github.com/Binaryify/OneDark-Pro/issues/736

This is clearly not intentional when you look at the OneDark Pro theme in VS Code, but after conversion, a bright white line is visible below tabs which is extremely distracting.

Is this something that may be optimised or improved in the converter?

Huge thanks
Fotis

**Comment by Binaryify** [comment]
@ketttY

**Comment by ketttY** [comment]
Hi @Binaryify and @fgimian, thank you for reporting this issue! I am wondering did you try converting with the latest theme converter? With the latest version, it should be using the `tab.activeBackground` for that divider, which should look like this: 
<img width="266" alt="image" src="https://user-images.githubusercontent.com/34032260/172448371-dde41125-0df6-4f5e-a8fe-7c8e7178e68d.png">
Wondering does this meet your expectations? Thank you!

**Comment by ketttY** [comment]
To make this behavior more stable (not rely on `tab.activeBorder` being null), I have created #146 to make this divider always use the color for `tab.inactiveBackground`.

**Comment by Binaryify** [comment]
> Hi @Binaryify and @fgimian, thank you for reporting this issue! I am wondering did you try converting with the latest theme converter? With the latest version, it should be using the `tab.activeBackground` for that divider (the original mapping is using `tab.activeBorder` but it falls back to `tab.activeBackground` because the active border color is null for One Dark Pro), which should look like this: <img alt="image" width="266" src="https://user-images.githubusercontent.com/34032260/172448371-dde41125-0df6-4f5e-a8fe-7c8e7178e68d.png"> Wondering does this meet your expectations? Thank you!

yes, I had used the latest theme converter, it is shows the bright line below tabs
After I use the dev/ketyang/tab branch, the bright line don't show now
<img width="780" alt="image" src="https://user-images.githubusercontent.com/12221718/172520801-ea8614b8-0410-4a0f-aa0e-ab7c3385760e.png">
but I think the border line should show in the bottom as same as vscode's
<img width="348" alt="image" src="https://user-images.githubusercontent.com/12221718/172520928-731ce39e-c1f5-4f18-8134-29673be30e03.png">

**Comment by ketttY** [comment]
> but I think the border line should show in the bottom as same as vscode's

Unfortunately, that is due to the design difference between VS and VSCode. For selected tabs at the top, the border will be `0,2,0,0` (which means only have a 2px wide border at the top) and the border brush is `FileTabSelectedBorderBrushKey`. So this is not an issue that the theme converter can fix.
